spec/max_sibling_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+require './max_sibling'
+
+describe MaxSibling do
+  describe '#solution' do
+    let(:instance) { described_class.new }
+
+    context 'given example' do
+      it 'should return 321 given 213' do
+        subject = instance.solution(213)
```

Comment:
We normally define `subject` after `describe` or `context`. https://relishapp.com/rspec/rspec-core/docs/subject/explicit-subject